modules/claims_api/app/sidekiq/claims_api/reporting_base.rb

```diff
@@ -142,14 +140,13 @@ def errored_ews
     end
 
     def map_transaction_ids(array)
-      # Dynamically generate unique keys like A, B, C, etc.
       transaction_mapping = {}
-      key_sequence = ('A'..'Z').to_a
+      key_sequence = (0..transaction_mapping.size + 3).to_a
```

Comment:
Iʼm not sure I understand - `transaction_mapping.size` will be `0` here, so we are setting `key_sequence` to `[0, 1, 2, 3]`? Wonʼt we run out of keys?

```
array = [
  { transaction_id: "A", id: 1 },
  { transaction_id: "B", id: 2 },
  { transaction_id: "A", id: 3 },
  { transaction_id: "C", id: 4 },
  { transaction_id: "D", id: 5 },
  { transaction_id: "E", id: 6 }
]

map_transaction_ids(array)
=>
{0=>[{:transaction_id=>"A", :id=>1}, {:transaction_id=>"A", :id=>3}],
 1=>[{:transaction_id=>"B", :id=>2}],
 2=>[{:transaction_id=>"C", :id=>4}],
 3=>[{:transaction_id=>"D", :id=>5}],
 nil=>[{:transaction_id=>"E", :id=>6}]}
```